Xplat/src/main/java/vazkii/botania/common/item/equipment/tool/manasteel/ManasteelPickaxeItem.java

```diff
@@ -72,11 +74,13 @@ public InteractionResult useOn(UseOnContext ctx) {
 					if (did.consumesAction() && !ctx.getLevel().isClientSide) {
 						ItemsRemainingRenderHandler.send(player, displayStack, TORCH_PATTERN);
 					}
-					return did;
+					if (did != InteractionResult.FAIL) {
```

Comment:
why not just did.isSuccess()? (or whatever it's called)

Reply:
I'm small brain